packages/node/src/lib/construct-payload.ts

```diff
@@ -96,6 +97,21 @@ export interface PayloadData {
   responseBody?: string;
 }
 
+/* This will generate an integrity hash that looks something like this:
+ *
+ * sha512-Naxska/M1INY/thefLQ49sExJ8E+89Q2bz/nC4Pet52iSRPtI9w3Cyg0QdZExt0uUbbnfMJZ0qTabiLJxw6Wrg==?1345
+ *
+ * With the last 4 digits on the end for us to use to identify it later in a list.
+ */
+export function mask(apiKey) {
+  return ssri
+    .fromData(apiKey, {
+      algorithms: ['sha512'],
+      options: [apiKey.slice(-4)],
```

Comment:
Thoughts on making this `slice` a percentage of overall characters? Like `5-10%`?

Reply:
We already spoke about this here: https://github.com/readmeio/metrics-sdks/pull/446/files#r879628785 

Opted to just do it like this for now for initial simplicity in implementation, but happy to revisit this in future.